packages/block-library/src/post-terms/block.json

```
@@ -27,7 +27,10 @@
 		"typography": {
 			"lineHeight": true,
 			"fontSize": true,
-			"__experimentalFontWeight": true
+			"__experimentalFontWeight": true,
```

It feels like this block should also have the extra features enabled. Maybe this just sticks out as the first block that really restricts which supports are opted into.

```suggestion
			"__experimentalFontStyle": true,
			"__experimentalFontWeight": true,
			"__experimentalLetterSpacing": true,
			"__experimentalTextTransform": true,
```